### ISSUE TITLE
Create Table for party committees that need to be distinguished beyond normal form 1 data

### ISSUE THREAD

**Issue by jwchumley** [fecgov/openFEC#3441]
Issue: Within party committees (committee type  in ('X','Y') ), there are some committees that need to be differentiated beyond normal committee type and designation. 

National Party Committees - These need to be identifiable for things like data summaries.
State Party Committees - These also need to be identifiable for separate treatement
Inaugural Committees - These file form 13 so the website front end needs to display them differently from form 3X filers 
Host Committees - These file form 4 so the website front end needs to display them differently from form 3X filers.
Convention Committees - These have no special display issues but should be able to be identified as separate from other committees for summary purposes.

Solution: Create a Party_type table that includes a column for Party_type as follows:
N - National Party Committee
S- State Party Committee
I - Inaugural Committee
K - Convention Committee
H - Host Committee

Update nightly process scripts so that form 1 and 1z don't change the org_type for committees in the table. to update the org_type with these codes for the committees in the table

**Comment by jwchumley** [comment]
I changed the Org type in all of the inaugural committee rows in the cmte_valid table expecting that to be included in the /committee/committee ID/history endpoint but I should have checked the underlying view first. It uses the form 1 table not the cmte_valid table. I've just set up a test for tonight to see if a change transaction in the form 1 table will do the job. If so then I'll move forward with the changes and with creating the table. 

Reminder - the table is for long term automation and is not necessary for any of the current issues that are blocked by this. Corretly updating the rows so that the API returns the correct org_type will suffice for unblocking.

**Comment by jwchumley** [comment]
Update: Last night's test worked. I've set all the inaugural committees up to process tonight and tomorrow the /committee/committee ID/history  will have organization type = 'I' so they can be differentiated as party committees filing form 13 instead of form 3x.

Now I'm looking at Host committees and might have them set up tonight as well.

**Comment by jwchumley** [comment]
Splitting out the table creation part of this ticket. It is on hold while I track how form 1 amendments affect the corrected rows in the database. The updated data part - this ticket - is closed.
